File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.cpp
class Solution {
public:
    int findMin(vector<int> &nums) {
        int st = 0 , end = nums.size()-1;
        while(st < end){
            int mid = (st+end)/2;
          if(nums[mid] > nums[nums.size()-1])
             st = mid + 1;
             else end = mid;
        }
        return nums[end];
    }
};

File: Data Structures & Algorithms/missing-number/submission-1.cs
public class Solution
{
    public int MissingNumber(int[] nums)
    {
        int cSum = 0;
        int sum = nums.Length * (nums.Length + 1)/2;
        for (int i = 0; i < nums.Length; i++)
        {
            cSum += nums[i];
        }
        return sum - cSum;
    }
}